packages/product-feed-persoo/tests/Unit/PersooFeedItemTest.php

```diff
@@ -0,0 +1,258 @@
+    {
+        $this->productUrlsBatchLoaderMock->method('getProductUrl')
+            ->with($product, $domain)->willReturn($url);
+    }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\Product $product
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domain
+     * @param string $url
+     */
+    private function mockProductImageUrl(Product $product, DomainConfig $domain, string $url): void
+    {
+        $this->productUrlsBatchLoaderMock->method('getProductImageUrl')
+            ->with($product, $domain)->willReturn($url);
+    }
+
+    public function testMinimalPersooFeedItemIsCreatable()
+    {
+        $persooFeedItem = $this->persooFeedItemFactory->create($this->defaultProduct, $this->defaultDomain);
+
+        self::assertInstanceOf(PersooFeedItem::class, $persooFeedItem);
+        self::assertEquals(self::PRODUCT_ID, $persooFeedItem->getId());
+        self::assertEquals(self::PRODUCT_ID, $persooFeedItem->getSeekId());
+        self::assertEquals(self::PRODUCT_NAME, $persooFeedItem->getTitle());
+        self::assertEquals(self::PRODUCT_URL, $persooFeedItem->getLink());
+        self::assertNull($persooFeedItem->getDescription());
+        self::assertEquals([0 => self::MAIN_CATEGORY_ID], $persooFeedItem->getHierarchyIdsIndexedByCategoryId());
+        self::assertEquals([0 => self::MAIN_CATEGORY_NAME], $persooFeedItem->getHierarchyNamesIndexedByCategoryId());
+        self::assertNull($persooFeedItem->getImageLink());
+        self::assertEquals(t('In stock'), $persooFeedItem->getAvailability());
+        self::assertThat($persooFeedItem->getPrice()->getPriceWithoutVat(), new IsMoneyEqual(Money::zero()));
+        self::assertThat($persooFeedItem->getPrice()->getPriceWithVat(), new IsMoneyEqual(Money::zero()));
+        self::assertEquals(self::EUR, $persooFeedItem->getCurrency()->getCode());
+        self::assertNull($persooFeedItem->getBrand());
+        self::assertEquals(['Flag_name'], $persooFeedItem->getFlagNames());
+        self::assertEquals([self::PARAMETER_NAME => self::PARAMETER_VALUE], $persooFeedItem->getProductParameterValuesIndexedByName());
+    }
+
+    public function testPersooFeedItemWithBrand()
+    {
+        /** @var \Shopsys\FrameworkBundle\Model\Product\Brand\Brand|\PHPUnit\Framework\MockObject\MockObject $brand */
+        $brand = $this->createMock(Brand::class);
+        $brand->method('getName')->willReturn(self::BRAND_NAME);
+        $this->defaultProduct->method('getBrand')->willReturn($brand);
+
+        $persooFeedItem = $this->persooFeedItemFactory->create($this->defaultProduct, $this->defaultDomain);
+
+        self::assertEquals(self::BRAND_NAME, $persooFeedItem->getBrand());
+    }
+
+    public function testPersooFeedItemWithDescription()
+    {
+        $this->defaultProduct->method('getDescription')
+            ->with(self::PRODUCT_ID)->willReturn(self::PRODUCT_DESCRIPTION);
```

Comment:
`getDescription` has `$domainId` as an argument, so there should be probably
```
->with(Domain::FIRST_DOMAIN_ID)
```

Reply:
It is not important if you use it only once

Reply:
yeah, it does not matter, it is just technically wrong :smile: :shrug:

Reply:
Changed